online-shop/src/app/product-list/product-list.component.scss

```diff
@@ -0,0 +1,35 @@
+table {
+  margin-right: auto;
+  margin-left: auto;
+  font-size: 1.25rem;
+  border: 1px solid black;
+  border-collapse: collapse;
+  text-align: center;
+  background-color: antiquewhite;
+}
+
+th {
+  border: 1px solid black;
+  width: 300px;
+  padding: 10px;
+}
+td {
+  width: 300px;
+border: 1px solid black;
```

Comment:
Don't forget about formatting and keeping a clean consistent code (HTML, TS and CSS/SCSS)